### DISCUSSION TITLE
Please add Align Your Steps scheduler.

### DISCUSSION THREAD

**Discussion by Shafinab03** [Acly/krita-ai-diffusion#691] [Ideas]
Request to add the Align your steps scheduler in the plugin. I am not sure if it is going to be hard to implement. It would be a major help because it boosts prompt adherence of sd1.5 tenfold.  The minimum step number of 10 steps also is fast enough to be almost instant

**Comment by rbfbatx** [comment]
Align Your Steps along with support for NVIDIA TensorRT would be awesome features to see in the plug in.

**Comment by Acly** [comment]
It's possible, but requires changing the sampler/scheduler node setup to something slightly more complex. 

> it boosts prompt adherence of sd1.5 tenfold

uh-huh

**Comment by Shafinab03** [comment]
> It's possible, but requires changing the sampler/scheduler node setup to something slightly more complex.

Is the "slightly" too much. for regular comfy it was as easy as just slapping the AYS node

**Comment by Acly** [comment]
Didn't mention here yet, but you can use use AYS now via [user sampler preset](https://github.com/Acly/krita-ai-diffusion/wiki/Samplers#creating-user-presets)

Put `align_your_steps` or `ays` as scheduler. Also check out `gits`, it's similar/improved version of AYS.
